README.md

```diff
@@ -144,17 +150,19 @@ It is time to setup the nodes of the cluster.
 
 Take a declarative approach and use [`terraform`](https://www.terraform.io) to setup the nodes of the cluster.
 Note that in this example I utilize the AWS Spots; you can easily change the code and use the regular on-demand instances.
-This is left as an exercise.
+
+This is left as an exercise. <!-- maybe give a hint here towards the solution or let people know that the solution is in the GitHub repo --->
```

Comment:
Well, the solution is not in the repo... Maybe rephrase to something like:

> Note that in this example I utilize the AWS Spots; it is possible to change the code and use the regular on-demand instances, but it is out of the scope of this tutorial.

In any case, it is an easy exercise :)